README.md

```diff
@@ -90,6 +90,38 @@ const PermissionExpandableRow = ({ dependencies: { experimentalFeatures } }) =>
 To make use of this feature by console it can be enabled like this:
 `localStorage.setItem('dopplerExperimental', JSON.stringify({PermissionHistory: true}));`
 
+## Guide Lines
+
+### Test
+
+- It's recommended to test from the user's point of view, hiding implementation details. [More details](https://testing-library.com/docs/).
+
+- Recommended documentation for [test implementation](https://github.com/FromDoppler/doppler-webapp/pull/1504).
+
+- Pre-commit hook is used in local development (husky+link-staged). [More details](https://github.com/FromDoppler/doppler-webapp/pull/1523).
+
+- There's no QA team, so each development must be tested. Also, if a new development is not tested, coverage decreases. If the coverage is below the accepted limit, the merge will not be possible. [More details](https://github.com/FromDoppler/doppler-webapp/pull/1593).
+
+### Code organization
+
+- **Components:** This is located in the *src/components* path. Each should be located in separate folders, each one must contain: *component implementation, tests, styles and images*. Also, the component is organized internally in the following order: hooks are located at the top, constants and states together in one place, next are the functions. Outside the component the variables or functions that do not depend on some props. [See slack conversation thread for more details](https://makingsense.slack.com/archives/CGJ72QFQX/p1625754680182000).
```

Comment:
I think it would be more clear this way:
```suggestion
- **Components:** This is located in the *src/components* path. Each should be located in separate folders, each one must contain: *component implementation, tests, styles and images*. Also, the component is organized internally in the following order:
- hooks are located at the top
- constants and states together
- functions.
**Outside the component** the variables or functions that do not depend on some props. [See slack conversation thread for more details](https://makingsense.slack.com/archives/CGJ72QFQX/p1625754680182000).
```

Reply:
```suggestion
- **Components:** This is located in the *src/components* path. Each should be located in separate folders, each one must contain: *component implementation, tests, styles and images*. Also, the component is organized internally in the following order:
  - hooks are located at the top
  - constants and states together
  - functions.
**Outside the component** the variables or functions that do not depend on some props. [See slack conversation thread for more details](https://makingsense.slack.com/archives/CGJ72QFQX/p1625754680182000).
```
I've forgot to add an extra level for bullets 😅

Reply:
Mmm, ahora mismo se está respetando todo esto?

Reply:
Bueno,  ahora mismo no veo múltiple niveles de identación.

Reply:
Estamos intentando ir hacia ahí, en cada PR es algo que vamos revisando, si hay que acomodar varios componentes que no respetan ese formato. A eso hacías referencia @andresmoschini ?